.changeset/plenty-readers-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/table': patch
+---
+
+Updating isHeader styles in Cell component
```

Comment:
We've technically added a prop tp the `Cell` component. This should be a `minor` bump, and we should add more details in the changelog